gnat2goto/driver/gnat_utils.adb

```diff
@@ -69,4 +70,33 @@ package body GNAT_Utils is
       end loop;
    end Iterate_Call_Parameters;
 
+   -------------------------------
+   -- Iterate_Pragma_Parameters --
+   -------------------------------
+
+   procedure Iterate_Pragma_Parameters (The_Pragma : Node_Id)
+   is
+      Params  : constant List_Id := Pragma_Argument_Associations (The_Pragma);
+      Param   : Node_Id := First (Params);
+      Arg_Pos : Positive := 1;
+   begin
+      pragma Assert (Is_Non_Empty_List (Params));
```

Comment:
this assert would be more meaningful just the `Params` declaration and before its first use